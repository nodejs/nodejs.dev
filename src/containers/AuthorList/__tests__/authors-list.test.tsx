import React from 'react';
import { render } from '@testing-library/react';
import AuthorsList from '..';

describe('AuthorsList component', () => {
  it('renders correctly', () => {
    const authors = ['test-author', 'another-test-author'];
    const { container } = render(<AuthorsList authors={authors} />);
    expect(container).toMatchSnapshot();
  });
});
