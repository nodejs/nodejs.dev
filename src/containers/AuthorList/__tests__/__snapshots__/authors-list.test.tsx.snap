// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthorsList component renders correctly 1`] = `
<div>
  <ul
    class="list"
  >
    <h5>
      Contributors
    </h5>
    <li>
      <a
        class="link"
        href="https://github.com/test-author"
        rel="noopener noreferrer"
        style="margin-left: 0px;"
        target="_blank"
        title="test-author"
      >
        <img
          alt="test-author"
          class="img"
          src="https://github.com/test-author.png?size=60"
        />
      </a>
    </li>
    <li>
      <a
        class="link"
        href="https://github.com/another-test-author"
        rel="noopener noreferrer"
        target="_blank"
        title="another-test-author"
      >
        <img
          alt="another-test-author"
          class="img"
          src="https://github.com/another-test-author.png?size=60"
        />
      </a>
    </li>
  </ul>
</div>
`;
